### DISCUSSION TITLE
'license' metadata is not emitted into Debian-style packages?

### DISCUSSION THREAD

**Discussion by kpfleming** [goreleaser/nfpm#713] [General]
I've just come across this tool today, but noticed that a `.deb` package created by it was marked as "Proprietary" while being installed using GNOME Software. After further investigation, it appears that the 'license' metadata field is not being emitted into the package. It does get emitted for RPMs, and probably other packages.

Is this just an oversight, or is there some limitation of the `.deb` format which would make this difficult/impractical to do?

**Comment by kpfleming** [comment]
It appears that #561 may be related, and the 'fix' might be some documentation telling users how to resolve this in their nFPM configuration.

**Reply by caarlos0** [reply]
ah, cool you figured it out... want to add it to the docs? the files are under the `www` folder in the repo.

**Reply by kpfleming** [reply]
Someone already did: https://nfpm.goreleaser.com/tips/#the-copyright-file

**Reply by pete-woods** [reply]
As this is the top hit for this search, the tip is now at: https://nfpm.goreleaser.com/docs/tips/#the-copyright-file

**Comment by Hadiza124** [comment]
Thank you for this tool fren
